setImmediate.js

```diff
@@ -172,4 +172,5 @@
 
     attachTo.setImmediate = setImmediate;
     attachTo.clearImmediate = clearImmediate;
-}(new Function("return this")()));
+}
+(typeof self == 'undefined' ? typeof global == 'undefined' ? this : global : self));
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>'global' is not defined.<br>'self' is not defined.